src/player.cpp

```diff
@@ -11325,6 +11360,31 @@ bool player::sees( const Creature &critter ) const
     return Creature::sees( critter );
 }
 
+float player::get_sees_modifier() const
+{
+    float modifier = 1.0;
+    if ( get_size() == MS_SMALL ) {
+        // player sillhouete is 50% smaller while sneaking
+        modifier = 0.5;
+    }
+
+    // having moved during last action adds 12.5% to 50% depending on movecost (slower is better)
+    if ( moved ) {
+        modifier += ( 0.25 / ( movecounter / 100.0 ) );
+    }
+
+    // survival skill helps with staying hidden in all circumstances (2.5% per level)
+    modifier -= get_skill_level( skill_survival ) * 0.025;
```

Comment:
Survival skill is already way too generalized and does everything at once. From foraging mushrooms, through being an edgy nihilist who feels no guilt for maiming undead humans, to extracting sci-fi gadgets from corpses.
Perception would be good here - if you can't see or hear yourself, you're either amazing at hiding or terrible at perception.